pymc4/tests/test_random_variables.py

```diff
@@ -41,7 +44,7 @@ def random_variable_args():
             {"loc": [1, 2], "covariance_matrix": [[0.36, 0.12], [0.12, 0.36]], "sample": [1, 2]},
         ),
         (random_variables.NegativeBinomial, {"total_count": 5, "probs": 0.5, "sample": 5}),
-        (random_variables.Normal, {"loc": 0, "scale": 1}),
+        (random_variables.Normal, {"loc": 0, "scale": 1, "sample": 0, "expected": [-.918]}),
```

Comment:
In regards to testing transforms, I set the test target here manually which isn't great. I think ideal case is to test against scipy, just like PyMC3. The problem though is that since the arg names are all different, everything would need to be restated.

If we do merge this PR one benefit would be being able to test against scipy easily
https://github.com/pymc-devs/pymc4/pull/70

Reply:
@canyon289 could you point me to the pm3 tests that you've referenced?

Reply:
Here's an example https://github.com/pymc-devs/pymc3/blob/master/pymc3/tests/test_distributions.py#L462

Reply:
@canyon289 I thought so too :smile:, was confused for a moment there.